frontend/src/components/camp/CampPrint.vue

```diff
@@ -5,16 +5,15 @@
       <v-btn color="primary" class="mt-5"
              :href="previewUrl"
              target="_blank">
-        Open print preview
+        {{ $tc('entity.camp.print.openPrintPreview') }}
       </v-btn>
       <v-btn
         color="primary"
         class="mt-5 ml-5"
         :loading="printing"
         @click="print">
-        Print now
+        {{ $tc('entity.camp.print.printNow') }}
```

Comment:
Same here, should be `components.camp.campPrint.printNow` (also remember to change it in the JSON)